freud/diffraction.pyx

```diff
@@ -0,0 +1,348 @@
+import numpy as np
+from scipy import interpolate, ndimage
+import rowan
+
+# from cme_utils.manip import pbc
+# from cme_utils.manip.utilities import rotation_matrix_from_to as cme_rot
+
+
+class Diffraction:
+    def __init__(
+        self,
+        grid_size=512,
+        zoom=4,
+        peak_width=1,
+        length_scale=3.905,
+        bot=4e-6,
+        top=0.7
+    ):
+        """
+        Initialize the diffractometer class
+        Parameters
+        ----------
+        grid_size : int, size of the diffraction grid (default 512)
+        zoom : (default 1)
+        peak_width : (default 1)
+        length_scale : (default 3.905)
+        bot : (default 4e-6)
+        top : (default 0.7)
+        """
+        self.N = grid_size
+        self.zoom = zoom
+        self.peak_width = peak_width
+        self.bin_w = 2.0
+        self.length_scale = length_scale
+        self.bot = bot
+        self.top = top
+
+    def load(self, system):
```

Comment:
Remove any methods that aren't used. I didn't see where this is used, at a quick glance.